docs/src/index.md

```diff
@@ -44,6 +44,9 @@ as in [dave-backend application.yml](https://github.com/it-at-m/dave-backend/blo
 ## Deploy
 
 DAVe __should be__ installed and operated via [helm chart](https://artifacthub.io/packages/helm/it-at-m/dave?modal=install).
-But for a basic peek or development environments on what DAVe has to offer, we provide a [docker-compose file](docker-compose.md).
 
+But for a basic peek or development environments on what DAVe has to offer, 
```

Comment:
Die Dockercomposebeschreibung in einen eigenen PR auslagern. Dieser PR wird mittlerweile für die Release-Notes verwendet.